docs/api/cookies.rst

```diff
@@ -9,10 +9,11 @@ Getting Cookies
 ~~~~~~~~~~~~~~~
 
 Cookies can be read from a request either via the
-:py:meth:`~.Request.get_cookie_values` method or the :py:attr:`~.Request.cookies`
-attribute on the :py:class:`~.Request` object. Generally speaking, the
-:py:meth:`~.Request.get_cookie_values` method should be used unless you need a
-collection of all the cookies in the request.
+:py:meth:`~.falcon.Request.get_cookie_values` method or the
```

Comment:
I just fixed these up for now to resolve doc warnings, but I plan to circle back in all these places and sort out how to also point people to ASGI Request/Response docs.